README.md

```diff
@@ -91,3 +91,13 @@ helm repo add giantswarm https://giantswarm.github.io/giantswarm-catalog
 helm repo update
 helm upgrade -i starboard-exporter --namespace <starboard namespace> giantswarm/starboard-exporter
 ```
+
+## One vulnerabilityreport per deployment
+
+By default starboard generates a vulnerabilityreport per replicaset in a deployment.
+This can cause confusion, especially if you fix a CVE in your latest deployment but the number of CVE:s per deployment stay the same in your metrics.
+
+In starboard v0.14.0 environment variable OPERATOR_VULNERABILITY_SCANNER_SCAN_ONLY_CURRENT_REVISIONS was introduced.
+Enabling this feature will only generate a vulnerabilityreport from the latest replicaset in a deployment.
+
+For more information on how to configure starboard look at the [docs](https://github.com/aquasecurity/starboard/blob/main/docs/operator/configuration.md).
```

Comment:
A few touch ups but otherwise LGTM. Can you also move this section up to line 55 (in the cardinality section)? I don't think it needs a separate heading, we can mention it alongside the other "lot of metrics" concerns.

```suggestion
By default, Starboard generates a `VulnerabilityReport` per ReplicaSet in a Deployment.
This can cause confusion because vulnerabilities are still reported for Pods which no longer exist, i.e. you fix a CVE in your latest Deployment but the number of CVEs per Deployment stays the same in your metrics.

As of Starboard v0.14.0, the environment variable `OPERATOR_VULNERABILITY_SCANNER_SCAN_ONLY_CURRENT_REVISIONS` can be enabled to only generate a `VulnerabilityReport` from the latest ReplicaSet in the Deployment.

Check the [Starboard configuration docs][starboard-config] for more information.
```

And then put this down by the other `starboard-upstream` markdown anchor:
```
[starboard-config]: https://github.com/aquasecurity/starboard/blob/main/docs/operator/configuration.md
```